ts/components/messages/MessageDetail/index.tsx

```diff
@@ -50,6 +76,29 @@ const OrganizationTitle = ({ name, organizationName, logoURLs }: UIService) => (
   />
 );
 
+const renderThirdPartyAttachmentsError = () => (
+  <>
+    <AttachmentsUnavailableComponent />
+    <View style={styles.viewFooter} />
```

Comment:
Maybe we can replace this `View` with a `VSpacer`

Reply:
The VSpacer's size does not permit to have the space spacing points that are outline in the FIGMA so the final visual effect is different

Reply:
what's the delta? we should stick to the component rules—the figma file is quite old

Reply:
32